src/Bundle/BundleExtension.php

```diff
@@ -2,71 +2,83 @@
 
 namespace Becklyn\RadBundle\Bundle;
 
+use Becklyn\RadBundles\Bundle\BundleExtension as NewBundleExtension;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\Container;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\Extension;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\Bundle\BundleInterface;
 
-/**
- * Base class to use in your bundle to easily create an extension.
- */
-class BundleExtension extends Extension
+if (\class_exists(NewBundleExtension::class))
 {
-    /** @var BundleInterface */
-    private $bundle;
-
-
-    /** @var string|null */
-    private $alias;
-
-
-    /**
-     */
-    public function __construct (
-        BundleInterface $bundle,
-        ?string $alias = null
-    )
+    class BundleExtension extends NewBundleExtension
```

Comment:
Both should be deprecated, as we still want to remove this calls and can't remove it, if they still use the name from RAD.